docs/adding_software/opening_pr.md

```diff
@@ -41,24 +41,34 @@ git remote add koala git@github.com:koala/software-layer.git
 ```
 
 3) Check out the branch that corresponds to the version of EESSI repository you want to add software to,
```

Comment:
This whole sentence should be rephrased, probably just to 'Make sure you're on the main branch:' (I think that's useful to keep, even if that's also the default branch - if people already cloned the repo earlier, it might help them remember that _at least_ they should switch back to the main branch).

Reply:
Changed in 24b7cfe.